lib/project_and_version_value.rb

```diff
@@ -61,9 +61,9 @@ def version_issues(proj_id, version_id)
   # @return [Issue] Two column,spent_on,sum of hours
   def version_costs(proj_id, version_id)
     proj = Project.find(proj_id)
-    Issue.cross_project_scope(proj, 'descendants')
-      .select('MAX(spent_on) AS spent_on, SUM(hours) AS sum_hours')
-      .where('start_date IS NOT NULL AND due_date IS NOT NULL AND fixed_version_id = ? ', version_id)
+    Issue.cross_project_scope(proj, "descendants")
+      .select("MAX(spent_on) AS spent_on, SUM(hours) AS sum_hours")
+      .where("start_date IS NOT NULL AND due_date IS NOT NULL AND fixed_version_id = ? ", version_id)
```

Comment:
Line is too long. [101/80]<br>Place the . on the previous line, together with the method call receiver.